### PR TITLE
feat: add `sbb-off-brand-colors` style

### PR DESCRIPTION
**Pull request by mhaertwig**
This style is intended to be used for security-critical applications. It replaces red colors with blue colors for all elements except error messages.

### CODE REVIEW

File: src/angular/typography.md [hunk 1]
```diff
@@ -19,6 +19,23 @@ To activate lean design, set the css class `sbb-lean` on your `<html class="sbb-
...
+except error messages. The style is based on the `sbb-lean` variant and can be activated
+by addmin the css class `sbb-off-brand-colors` to your page's `<html>` element.
+
+```html
+<html class="sbb-lean sbb-off-brand-colors"></html>
+```
+
+For viewing this showcase in the `sbb-off-brand-colors` mode you can simply click
+[this link](/?mode=sbb-off-brand-colors) or add the `?mode=sbb-off-brand-colors` query parameter
+to any pageof this showcase.
```
Comment (jeripeierSBB) [inline src/angular/typography.md:37]:
```suggestion
to any page of this showcase.
```

File: src/showcase/styles.scss [hunk 2]
```diff
@@ -25,6 +25,10 @@ sbb-how-to-update {
   h1 {
     margin-top: 1rem !important;
     color: #eb0000;
```
Comment (jeripeierSBB) [inline src/showcase/styles.scss:27]: var verwenden

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Review by jeripeierSBB — APPROVED** [review]
thanks, LGTM

File: src/showcase/app/shared/mode-notification-toast/mode-notification-toast.component.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+const modeLocalstorageKey = 'sbbAngularMode';
+
+@Component({
+  selector: 'sbb-mode-notification-toast',
+  template: `The app is currenly shown in the "sbb-off-brand-colors" mode.
```
Comment (mhaertwig) [inline src/showcase/app/shared/mode-notification-toast/mode-notification-toast.component.ts:8]: ViewEncapsulaion.None

File: src/showcase/app/app-routing.module.ts [hunk 4]
```diff
@@ -20,15 +20,25 @@ const routes: Routes = [
           sessionStorage.setItem(modeSessionStorageKey, offBrandColorMode);
         }
 
+        let variantPrimaryColor;
         if (sessionStorage.getItem(modeSessionStorageKey) === offBrandColorMode) {
+          variantPrimaryColor = '#06348b';
```
Comment (jeripeierSBB) [inline src/showcase/app/app-routing.module.ts:25]: Do variables not work here? Or could you at least parse the variable by getComputedStyle?

Comment (kyubisation) [reply src/showcase/app/app-routing.module.ts:25]: `getComputedStyle(document.documentElement).getPropertyValue('--sbb-color-red')` could be used after the if/else.

File: src/angular/header-lean/header.scss [hunk 5]
```diff
@@ -214,6 +214,10 @@
 
   > svg {
     width: calc(#{sbb.pxToRem(80)} * var(--sbb-scaling-factor));
+
+    :where(.sbb-off-brand-colors) & path:first-child {
```
Comment (kyubisation) [inline src/angular/header-lean/header.scss:218]: Should we maybe add a class to our default logo and add it to the selector? This way, if a consumer uses a different svg it does not have side effects.

File: src/showcase/app/app-routing.module.ts (hunk 4 above)
Comment (mhaertwig) [reply src/showcase/app/app-routing.module.ts]: Thanks for the hint. I fixed it

**Pull request merged**